File: customers-staff-master/src/Front/src/api/new/dto/ServiceCalendar/ServiceCalendarRecordDto.ts
import { Guid } from "../Guid";
import { CustomerStatusDto, RecordStatusDto, ServiceCalendarRecordInfoDto } from "./ServiceCalendarRecordInfoDto";

export interface ServiceCalendarRecordDto extends ServiceCalendarRecordInfoDto {
  id: Guid;
  recordStatus: RecordStatusDto;
  customerStatus: CustomerStatusDto;
}

File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/Calendars/ServiceCalendarRecordDto.cs
using System;

namespace Market.CustomersAndStaff.FrontApi.Dto.Calendars
{
    public class ServiceCalendarRecordDto : ServiceCalendarRecordInfoDto
    {
        public Guid Id { get; set; }
        public RecordStatusDto RecordStatus { get; set; }
        public CustomerStatusDto CustomerStatus { get; set; }
    }
}